### PR TITLE
bugfix: ZENKO-1908 update sproxydclient

### PR DESCRIPTION
**Pull request by alexanderchan-scality**

### CODE REVIEW

**Review by jonathan-gramain — APPROVED** [review]

**Review by nicolas2bert — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "simple-glob": "^0.2.0",
     "socket.io": "~2.2.0",
     "socket.io-client": "~2.2.0",
-    "sproxydclient": "scality/sproxydclient#45090b7",
+    "sproxydclient": "github:scality/sproxydclient#a6ec980",
```
Comment (bennettbuchanan) [inline package.json:43]: I've seen this prefix `github:` added when running `npm i`. It sounds harmless, but wondering if there's a reason we seem to consistently remove it from our package.json.

**Review by JianqinWang — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (rahulreddy) [reply package.json:43]: It's npm inconsistency...some versions add, some don't.

**Review by bennettbuchanan — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (bennettbuchanan) [reply package.json:43]: As long as we're compatible LGTM.

**Comment by alexanderchan-scality** [comment]
@bert-e approve

**Pull request merged**